app/models/event_topic.py

```diff
@@ -22,17 +22,22 @@ class EventTopic(db.Model):
     name = db.Column(db.String, nullable=False)
     system_image_url = db.Column(
         db.String, nullable=False,
-        default='https://image.ibb.co/iTrd5y/system_image_common.png'
+        default='http://localhost:5000/static/header.png'
```

Comment:
Not really good. If it is hosted on heroku, this should not point to localhost

Reply:
@bhaveshAn can you maybe use the `SERVER_NAME` or use something like `url_for`